plugins/available/killrails.plugin.bash

```diff
@@ -0,0 +1,15 @@
+cite about-plugin 
+about-plugin 'Function to kill a daemonized Rails server.'
+
+# Quick function to kill a daemonized Rails server
+function killrails() {
+  railsPid="$(cat tmp/pids/server.pid)"
+  if [ ! -z "$railsPid" ]; then
+    echo "[OK] Rails Server Process Id : ${railsPid}"
+	kill -9 $railsPid
```

Comment:
Indentation still looks wonky from this line on - please use 2 spaces for each level. Maybe enable displaying whitespace characters in your editor to make this easier...